nikola/data/themes/bootstrap3/templates/base.tmpl

```diff
@@ -75,14 +75,22 @@ ${template_hooks['extra_head']()}
 </div>
 
 ${base.late_load_js()}
-    <script>$('a.image-reference:not(.islink) img:not(.islink)').parent().colorbox({rel:"gal",maxWidth:"100%",maxHeight:"100%",scalePhotos:true});</script>
+    % if use_cdn:
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/baguettebox.js/1.8.2/baguetteBox.min.js"></script>
+    % else:
+        <script src="/assets/js/baguetteBox.min.js"></script>
+    % endif
     <!-- fancy dates -->
     <script>
     moment.locale("${momentjs_locales[lang]}");
     fancydates(${date_fanciness}, ${js_date_format});
     </script>
     <!-- end fancy dates -->
     <%block name="extra_js"></%block>
+    <script>baguetteBox.run('a.reference', {
```

Comment:
Why did we change the classes baguetteBox works with?

Reply:
That's copied from base theme. Probably both should be what was in bootstrap instead.

I'll take a look at it tomorrow.

Reply:
Ping?

Reply:
This seems to work in the demo cases. Since baguette is not JQuery based, I don't see a simple way to use that complex selector :-(